.vscode/docker-compose.debug.yml

```diff
@@ -0,0 +1,75 @@
+# docker-compose for debugging with vscode
+# docker-compose -p hydroshare -f .vscode/docker-compose.debug.yml up
+version: '3'
+services:
+  data.local.org:
+    extends:
+      file: ../local-dev.yml
+      service: data.local.org
+  users.local.org:
+    extends:
+      file: ../local-dev.yml
+      service: users.local.org
+  postgis:
+    extends:
+      file: ../local-dev.yml
+      service: postgis
+  rabbitmq:
+    extends:
+      file: ../local-dev.yml
+      service: rabbitmq
+  solr:
+    extends:
+      file: ../local-dev.yml
+      service: solr
+    volumes:
+      - "..:/hydroshare"
+      - "solr_data_vol:/opt/solr/server/solr"
+  nginx:
+    extends:
+      file: ../local-dev.yml
+      service: nginx
+    build:
+      context: ../nginx
+      dockerfile: Dockerfile-nginx
+  hydroshare:
+    extends:
+      file: ../local-dev.yml
+      service: hydroshare
+    build:
+      context: ..
+      dockerfile: Dockerfile
+    volumes:
+      # hydroshare repository
+      - "..:/hydroshare"
+      # shared location for gunicorn.sock between containers
+      - "temp_vol:/hs_tmp"
+      # temp directory shared with celery workers
+      - "share_vol:/shared_tmp"
+    ports:
+      - "1338:2022"
+      - "8000:8000"
+      - 5678:5678
```

Comment:
@sblack-usu only 2 lines modified. Opened a new port for the pydebug to listen on, and call `.vscode/init-hydroshare-debug` instead of `init-hydroshare`